sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -355,42 +356,17 @@ export class JsonRpcProvider extends Provider {
   }
 
   async getTransactionsForObject(
```

Comment:
I think these APIs really need some love, passing argument lists like this doesn't scale super well, we should probably move to objects for additional request metadata at some point.